.circleci/config.yml

```diff
@@ -6,13 +6,13 @@ orbs:
 executors:
   nodejs-browsers:
     docker:
-      - image: votingworks/cimg-debian11-browsers:2.0.0
+      - image: votingworks/cimg-debian11-browsers:2.0.2
```

Comment:
Is there a commit/PR associated with this version bump?

Reply:
https://github.com/votingworks/vxsuite-build-system/pull/19